### PR TITLE
Migration to 11 from 8 - Base import log and sales order import modules

### PR DESCRIPTION
**Pull request by mra-odoo**
Requirement:

Our new customer will be using Odoo v11.0 and we would like have two modules in v8.0 to be migrated to v11.0. The following is the links to the modules:
https://github.com/rfhk/she-custom/tree/master/sale_order_import
https://github.com/rfhk/she-custom/tree/master/base_import_log

Done with this PR

Thanks,
Mustufa R

### CODE REVIEW

File: sale_order_import/wizard/import_sale.py [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+                        #IF INVOICE POLICY IS FROM DELIVERY ORDER THEN WE WILL NOT CREATE INVOICE FROM HERE AND WE WILL PROCESS/CREATE INVOICE WHILE IMPORTING PICKINGS CSV. MEANS PICKING WILL BE HAVING STATE 2BINVOICED WILL BE PROCESS INVOICE/PAYMENT WHILE IMPORTING PICKING CSV.
+                        #invoice = picking.action_invoice_create(
+                        #                        journal_id = self.customer_invoice_journal_id.id,
+                        #                        type = 'out_invoice'
+                        #                        )
+
+#                     if order_id.state == 'manual' or order_id.state == 'prepaid':
+#                         invoice = order_id.signal_workflow('manual_invoice') # odoo 11
+
+                    invoice_ids = order_id.action_invoice_create() # odoo11
```
Comment (TimLai125) [inline sale_order_import/wizard/import_sale.py]: The import is not success since qty_to_invoice in all sale.order.line of the sales order are always 0.0.

Comment (TimLai125) [reply sale_order_import/wizard/import_sale.py]: I have checked the details. After setting the invoice policy of the products, the import is successful.

**Comment by TimLai125** [comment]
@yostashiro Please review the PR, the module should be good to be merged.

**Comment by yostashiro** [comment]
Taken over by https://github.com/rfhk/sts-custom/pull/9.  Closing this for now.

**Pull request closed without merging**